### PR TITLE
Yanwen Wang hw01

### PR DESCRIPTION
**Pull request by yanwenwang24**
Yanwen Wang hw01

Experience:
- Markdown is easy, since I have some previous experience using Jupyter Notebook in Python.
- Commit and push are new concepts to me.

Difficulties:
- Setting the correct path for the image. I initially set the path according to RStudio cloud/project/image, but instead I should follow the directory of github.
- I did some research about how to configure RStudio Cloud and connect it to Github.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* Great job on completed all the requirements for the HW :)
* For the future note, your workflow and difficulties should be in the comment when you pull request.
* I play piano too! I love Schubert's Piano Sonata in A minor.

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* Great job on completed all the requirements for the HW :)
* For the future note, your workflow and difficulties should be in the comment when you pull request.
* I play piano too! I love Schubert's Piano Sonata in A minor.

**Review by medcar8879 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity**  |    X       |              |            |
| **Ease of access**          |      X     |              |            |

Very nicely done! I'm impressed by all of your formatting, it looks great :) My two comments are that your own comments about workflow and difficulties can be added as a comment on your pull request rather than in the text of the assignment itself. Also, small grammatical thing, but "followings" should instead be "following." Again, great job!

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+## Yanwen Wang
+I'm **Yanwen Wang**, a second-year student at UChicago SSA. Here's my [Linkedin](https://www.linkedin.com/in/yanwen-wang-3a0329105/) page. The followings are a list of my hobbies:
+
+- _Piano:_ My favorite piece to play is "The Goldberg Variations".
```
Comment (wanitchayap) [inline README.md:5]: Great syntax :)

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+## Images
+![Image of Yanwen](yanwen.jpeg)
+
+## General workflow
+I used RStudio Cloud on iPad since my laptop is under repair. Basically, after forking the project, I opened a new project with a Git Repo on RStudio Cloud, thus connecting my RStudio to Github. I made changes to the Readme.md using RMarkdown. With some previous experience using Jupyter Notebook with Python, I didn't find any difficulty when using RMarkdown. And now I'm ready to make the push request and upload it back to Github.
+
+## Difficulties
+I experienced the most difficulties during the commit and push procedure, expecially wondering how to push the md file along with an image in the working directory.
+
+Initial mistake: I set up the path of the image as /cloud/project/image, according to the directory in Rstudio, however, I should directly set the path according to Github directory.
```
Comment (wanitchayap) [inline README.md:18]: just a cool tweak you can do here:
`/cloud/project/image`
Use `` to format as a code!

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+# hw01
+## Yanwen Wang
+I'm **Yanwen Wang**, a second-year student at UChicago SSA. Here's my [Linkedin](https://www.linkedin.com/in/yanwen-wang-3a0329105/) page. The followings are a list of my hobbies:
+
+- _Piano:_ My favorite piece to play is "The Goldberg Variations".
+- _Photography:_ My photography style is very conceptual.
+- _Hiking:_ I summitted Mt. Kilimanjaro in 2017.
+
+## Images
+![Image of Yanwen](yanwen.jpeg)
```
Comment (boruis12) [inline README.md:10]: Cute kitten!

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+## Yanwen Wang
+I'm **Yanwen Wang**, a second-year student at UChicago SSA. Here's my [Linkedin](https://www.linkedin.com/in/yanwen-wang-3a0329105/) page. The followings are a list of my hobbies:
```
Comment (boruis12) [inline README.md:3]: nice use of header, link, list and emphasis  in one section

**Review by boruis12 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |


Remarks:
Great introduction of yourself and explanation of your workflow! 
Your image loaded properly and the link to your LinkedIn profile worked.
Nice use of difference levels of header. 
After reading your hw01, I learnt that Rstudio Cloud can be used on iPad. 
Hope after repair, our laptop is brand as new!

**Pull request closed without merging**